zerver/webhooks/trello/fixtures/update_check_item.json

```diff
@@ -0,0 +1,5 @@
+{
+    "action": {
+        "type": "updateCheckItem"
+    }
+}
```

Comment:
The actual webhook contents are a bit more involved than this.  I think we want to have a more accurate fixture, especially since `create_check_item.json` and other ignored hooks do so.

Reply:
I think having full fixtures for payloads that we just ignore is just gonna be noise.